.changelog/3064.feature.md

```diff
@@ -0,0 +1,40 @@
+go/oasis-node: CLI support for the GetStatus endpoint
+
+Run
+
+```sh
+oasis-node control status
+```
+
+to get information like the following:
```

Comment:
Instead of having this long section in the changelog, consider moving this somewhere under `docs/oasis-node` (maybe `docs/oasis-node/cli.md`, section "Node Status" or something similar)? Then link the documentation from the changelog entry.

Reply:
ok

Reply:
I ended up using subcommand names for the section names